hydromt_wflow/wflow_sediment.py

```diff
@@ -244,88 +250,63 @@ def setup_outlets(
+        This function directly calls the ``setup_gauges`` function of the WflowModel,
+        see py:meth:`hydromt_wflow.wflow.WflowModel.setup_gauges` for more details.
 
-        Adds model layers:
+        The only differences are the default values for the arguments:
 
-        * **wflow_gauges_source** map: gauge IDs map from source [-] (if gauges_fn)
-        * **wflow_subcatch_source** map: subcatchment based on gauge locations [-] \
-(if derive_subcatch)
-        * **gauges_source** geom: polygon of gauges from source
-        * **subcatch_source** geom: polygon of subcatchment based on \
-gauge locations [-] (if derive_subcatch)
+        - ``gauge_toml_header`` defaults to ["Q", "TSS"]
+        - ``gauge_toml_param`` defaults to ["lateral.river.q_riv",
+            "lateral.river.SSconc"]
 
-        Parameters
-        ----------
-        gauges_fn : str, {"grdc"}, optional
-            Known source name or path to gauges file geometry file, by default None.
-        source_gdf : geopandas.GeoDataFame, optional
-            Direct gauges file geometry, by default None.
-        snap_to_river : bool, optional
-            Snap point locations to the closest downstream river cell, by default True
-        mask : np.boolean, optional
-            If provided snaps to the mask, else snaps to the river (default).
-        derive_subcatch : bool, optional
-            Derive subcatch map for gauges, by default False
-        derive_outlet : bool, optional
-            Derive gaugemap based on catchment outlets, by default True
-        basename : str, optional
-            Map name in grid (wflow_gauges_basename).
-            If None use the gauges_fn basename.
-        toml_output : str, optional
-            One of ['csv', 'netcdf', None] to update [csv] or [netcdf] section of \
-wflow toml file or do nothing.
-            By default, 'csv'.
-        gauge_toml_header : list, optional
-            Save specific model parameters in csv section.
-            This option defines the header of the csv file.
-            By default saves Q (for lateral.river.q_riv) and
-            TSS (for lateral.river.SSconc).
-        gauge_toml_param: list, optional
-            Save specific model parameters in csv section.
-            This option defines the wflow variable corresponding to the
-            names in gauge_toml_header.
-            By default saves lateral.river.q_riv (for Q) and
-            lateral.river.SSconc (for TSS).
+        See Also
+        --------
+        WflowModel.setup_gauges
         """
         # # Add new outputcsv section in the config
         super().setup_gauges(
             gauges_fn=gauges_fn,
-            source_gdf=source_gdf,
+            index_col=index_col,
             snap_to_river=snap_to_river,
             mask=mask,
+            snap_uparea=snap_uparea,
+            max_dist=max_dist,
+            wdw=wdw,
+            rel_error=rel_error,
             derive_subcatch=derive_subcatch,
```

Comment:
```suggestion
            abs_error=abs_error,
            fillna=fillna,
            derive_subcatch=derive_subcatch,
```

Also add the arguments here.